src/backend/cdb/cdbpath.c

```diff
@@ -885,6 +888,11 @@ cdbpath_distkeys_from_preds(PlannerInfo *root,
  * mergeclause_list is a List of RestrictInfo.  Its members are
  * the equijoin predicates between the outer and inner rel.
  * It comes from select_mergejoin_clauses() in joinpath.c.
+ * 
+ * Besides ordering logic, 
```

Comment:
Extra spaces were appended to two lines above

Reply:
Resolved